Cargo.toml

```diff
@@ -155,3 +155,7 @@ xcm = { git = "https://github.com/paritytech//polkadot", branch = "release-v0.9.
 [patch.crates-io]
 sp-core = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.42" }
 sp-runtime = { git = "https://github.com/paritytech//substrate", branch = "polkadot-v0.9.42" }
+
+# We need to patch this to https://github.com/tkaitchuck/aHash/releases/tag/v0.8.11 to prevent a build error
+# 'error[E0635]: unknown feature `stdsimd`' that occurs because this feature was removed in the latest nightly versions
+ahash = { git = "https://github.com/tkaitchuck/aHash", rev = "db36e4c4f0606b786bc617eefaffbe4ae9100762" }
```

Comment:
I also [experienced this](https://github.com/pendulum-chain/pendulum/pull/463#issue-2291953481), that's why I moved up to `nightly-2024-04-18`. :pensive:

Pendulum has 2 ahash dependencies:
https://github.com/pendulum-chain/pendulum/pull/463/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eR150-R172

Reply:
Do you think `nightly-2024-04-18` would work in Spacewalk too? Or was there another issue? I don't remember.

Reply:
Should we define `nightly-2024-04-18` in the rust-toolchain file then and remove this patch for `ahash` from the `Cargo.toml` file? Or what do you think @b-yap?

Reply:
@b-yap any thoughts on this?

Reply:
mmm, I think the patch isn't needed?
I did a `cargo update` in my previous PR; the [`ahash` in cargo.lock should be ok for now](https://github.com/pendulum-chain/pendulum/pull/463/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eL150-L170). 
I think I mentioned the minimum nightly version in the readme; but I did not explain why. 
https://github.com/pendulum-chain/pendulum?tab=readme-ov-file#how-to-run-tests 
We could add the reason over there.

Reply:
I removed the patch statement again and updated all references (also in the CI file) to point to the new nightly version. Let's see if the CI passed and then I merge.